required/examples/basics/video.js

```diff
@@ -1,26 +1,13 @@
-var renderer = PIXI.autoDetectRenderer(800, 600, { transparent: true });
-document.body.appendChild(renderer.view);
-
-// create the root of the scene graph
-var stage = new PIXI.Container();
+var app = new PIXI.Application(800, 600, { transparent: true });
+document.body.appendChild(app.view);
 
 // create a video texture from a path
 var texture = PIXI.Texture.fromVideo('required/assets/testVideo.mp4');
```

Comment:
May want to require a 'play'touch, which goes through baseTexture to call play on html video so that this works on iPhones